src/modules/mavlink/mavlink_receiver.cpp

```diff
@@ -1516,9 +1516,9 @@ void MavlinkReceiver::fill_thrust(float *thrust_body_array, uint8_t vehicle_type
 
 	case MAV_TYPE_VTOL_DUOROTOR:
 	case MAV_TYPE_VTOL_QUADROTOR:
+	case MAV_TYPE_VTOL_FIXEDROTOR:
```

Comment:
Where did these changes come in?

Reply:
It's cherry picked from here, to fix compile issues due to changes in Mavlink submodule!

https://github.com/PX4/PX4-Autopilot/pull/19375